tensorflow/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/GraphTrainableModel.kt

```diff
@@ -825,20 +825,17 @@ public abstract class GraphTrainableModel(vararg layers: Layer) : TrainableModel
         when (writingMode) {
             WritingMode.FAIL_IF_EXISTS -> {
                 check(!modelDirectory.exists()) { "The directory exists on path $pathToModelDirectory, please be careful it could contain valuable model! Change this mode to OVERRIDE if you want to override this directory." }
-                Files.createDirectories(modelDirectory.toPath())
```

Comment:
I need to say, that this is not the same code with replacement of createDirectories + mkdir => mkdirs due to reducing number of checks and supported exceptions

But I agree, it should be consistent in both files

Reply:
Wow, really, double creation of the last directory, ok, I will check and return